### PR TITLE
Fix BatchNorm2d test using inputs of wrong number of dimensions

### PR DESCRIPTION
**Pull request by SsnL**
BatchNorm2d should see 4d inputs. The onnx-fb-universe test in https://github.com/pytorch/pytorch/pull/4922 fails because that PR restores dimensionality check in BN modules.

### CODE REVIEW

**Comment by bddppq** [comment]
@onnxbot test this please

**Comment by SsnL** [comment]
Looking at the test failure, I'm not really understanding this line:
```
19:17:41     name: "torch-jit-export"
19:17:41     initializer {
19:17:41       dims: 2
19:17:41       data_type: FLOAT
19:17:41       name: "1"
19:17:41 -     raw_data: "\217~,?b\265\251>"
19:17:41 +     raw_data: "\330=\221>|\037(?"
19:17:41     }
```

The other differences are expected.

**Comment by houseroad** [comment]
The initializer changed. So please use the --accept flag to update expect files.

**Comment by bddppq** [comment]
@onnxbot retest this please

**Comment by bddppq** [comment]
@SsnL Thanks!
